microsoft-edge/web-platform/translator-api.md

```diff
@@ -0,0 +1,431 @@
+---
+title: Translate text with the Translator API
+description: Translate text with the Translator API
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.date: 09/05/2025
+---
+# Translate text with the Translator API
+<!-- todo: create a short link like https://aka.ms/edge-translator-api-docs -->
+
+The Translator API is an experimental web API that allows you to translate text between two different languages by using a small language model (SLM) that is built into Microsoft Edge, from JavaScript code in your website or browser extension.
+
+For introductory information, see [Translator and Language Detector APIs](https://webmachinelearning.github.io/translation-api/).
+
+
+**Detailed contents:**
+
+* [Availability of the Translator API](#availability-of-the-translator-api)
+* [Alternatives to and benefits of the Translator API](#alternatives-to-and-benefits-of-the-translator-api)
+* [The Phi-4-mini model](#the-phi-4-mini-model)
+    * [Disclaimer](#disclaimer)
+    * [Hardware requirements](#hardware-requirements)
+    * [Model availability](#model-availability)
+* [Enable the Translator API](#enable-the-translator-api)
+* [See a working example](#see-a-working-example)
+* [Use the Translator API](#use-the-translator-api)
+    * [Check if the API is enabled](#check-if-the-api-is-enabled)
+    * [Check if the model can be used](#check-if-the-model-can-be-used)
+    * [Create a new session](#create-a-new-session)
+        * [Monitor the progress of the model download](#monitor-the-progress-of-the-model-download)
+    * [Run the Translator API](#run-the-translator-api)
+        * [Wait for the text to be translated](#wait-for-the-text-to-be-translated)
+        * [Display tokens as they are generated](#display-tokens-as-they-are-generated)
+    * [Stop translating text](#stop-translating-text)
+    * [Destroy a session](#destroy-a-session)
+        * [Destroy a session by using the destroy() method](#destroy-a-session-by-using-the-destroy-method)
+        * [Destroy a session by using an AbortController](#destroy-a-session-by-using-an-abortcontroller)
+* [Send feedback](#send-feedback)
+* [See also](#see-also)
+
+
+<!-- ====================================================================== -->
+## Availability of the Translator API
+
+The Translator API is available as a developer preview in the Microsoft Edge Canary or Dev channels, starting with version 1234<!-- todo: fix version # -->.
+
+The Translator API is optimized for translating text content.
+
+As an alternative, the Prompt API serves more custom prompt-engineering scenarios; see [Prompt a built-in language model with the Prompt API](./prompt-api.md).
```

Comment:
move paragraph down to "Alternatives to and benefits of the Translator API"?

@captainbrosset